client/src/components/RichTextEditor.js

```diff
@@ -64,11 +82,44 @@ const RichTextEditor = ({
       editor.children = createSlateValue(value)
       editor.onChange()
     }
-  }, [editor, previousValue, readOnly, value])
+  }, [editor, previousValue, readOnly, disableFullSize, value])
 
   const renderElement = useCallback(props => <Element {...props} />, [])
   const renderLeaf = useCallback(props => <Leaf {...props} />, [])
 
+  useEffect(() => {
+    editableRef.current = ReactEditor.toDOMNode(editor, editor)
+  }, [editor])
+
+  const handleFullSizeMode = isFullSize => setShowFullSize(isFullSize)
+
+  const makeToolbarAccessible = debounce(node => {
+    if (showFullSize || !Range.isRange(node.selection)) {
+      return
+    }
+    const toolbarRect = toolbarRef.current.getBoundingClientRect()
```

Comment:
May throw the following error:

![image](https://user-images.githubusercontent.com/45197818/229137156-77d4f0c3-eec2-48d9-8ca0-268e77afc949.png)